### DISCUSSION TITLE
Any thoughts/interest about integration with Quarto?

### DISCUSSION THREAD

**Discussion by kjohnsen** [fonsp/Pluto.jl#2258] [Pluto development]
Another one of those 'this would be really cool but I don't have time/expertise to offer to actually make it happen' ideas 😆 

I recently came across this new [Quarto](https://quarto.org/) thing. Looks pretty nifty. They support Jupyter notebooks (Python AND Julia), RMarkdown, and Observable JS. If they already support one reactive kernel (Observable), how cool would it be to have reactive Julia through Pluto?

**Comment by rikhuijzer** [comment]
How would an end result look? Basically, generating HTML, PDF, MS Word, ePub etc from Pluto notebooks with cross references? See also https://github.com/fonsp/Pluto.jl/discussions/2106.

I think it's probably safer to then write native Julia code for HTML and PDF generation. No need to be restricted by Quarto's system since they don't add that much value. Generating Word documents is terrible anyway, the formatting of tables and images is poor. Also, Pandoc is a risky dependency for lots of business users because of the GPLv2 license.

**Reply by kjohnsen** [reply]
Hmm...just trying to make sense of this landscape of tools...from what I understand, Quarto basically provides an alternative to Books.jl or Weave.jl and it looks like you're working on integrating PlutoStaticHTML with Books. So what does Quarto do that Books can't? I guess it would be reveal.js presentations and LaTeX for journal articles.

And a newbie question about GPLv2--would it be risky for businesses because any modified code couldn't be made proprietary?

**Reply by rikhuijzer** [reply]
Books.jl is also very fixed in how the site looks so probably differences are there too

I donno about GPL. I only heard that some people in companies don’t like it
